sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -50,7 +58,7 @@ public class EventData
         ///
         public Stream BodyAsStream
```

Comment:
nit:  No need for the `get` here either _(but GH won't let me make a suggestion....)_

```csharp
public Stream BodyAsStream => BodyAsBinaryData.ToStream();
```

Reply:
Updated both this and Body to use expression-bodied property declaration.